### DISCUSSION TITLE
[ONLINE] 43 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#96] [Wydarzenia]
Podczas najbliższego spotkania będziemy mieli przyjemność posłuchać dwóch niezwykle interesujących sesji.
Jako pierwszy wystąpi nasz **gość specjalny - Thomas Maurer** - wieloletni MVP, a obecnie pracownik Microsoft. Drugi na scenie pojawi się **Patryk Lotzwi, zwycięzca głosowania** w tym miesiącu (116 głosów) z sesją o Azure DevOps.

Agenda:
18:00-18:05 - Powitanie
18:05-19:05 - "Manage and govern your hybrid servers using Azure Arc" - Thomas Maurer @thomasmaurer
19:05-20:05 - "Multi-stage pipelines in Azure DevOps" - Patryk Lotzwi @patrikulus

Więcej szczegółów tutaj: https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/273449464/

**Comment by kaluzaaa** [comment]
Przypominamy to już dziś‼️

**Reply by pstolarczyk** [reply]
Hej, będzie udostępniona sesja Patryka? - nie mogłem być do końca

**Reply by patrikulus** [reply]
Z tego co widzę, to nagranie z całego spotkania jest dostępne ;) https://www.youtube.com/watch?v=rnAGuIz7HPo

**Reply by pstolarczyk** [reply]
no tak 👍 ;) dzięki
